templates/advancement/size-flow.hbs

```diff
@@ -0,0 +1,9 @@
+<form id="{{appId}}" data-level="{{level}}" data-id="{{advancement.id}}" data-type="{{type}}">
+  <h3>{{{this.title}}}</h3>
+
+  <p>{{advancement.configuration.hint}}</p>
+
+  <select name="size">
+    {{selectOptions sizes selected=selectedSize}}
+  </select>
```

Comment:
I think it's a little confusing to be presented with a drop-down that has only one option. A user might reasonably think there was some bug and that they were supposed to have a choice. One option that might work is to only show the drop-down if `sizes.length > 1`, and just display the hint otherwise. In your Elf example, the hint very clearly tells the user that their size is Medium. If the hint text is omitted, I think we can quite easily auto-generate some hint text like we do with traits that says something like 'Your size is {size}' in the case of `sizes.length < 2` and 'Choose one of the available options for your size' if `sizes.length > 1`.